src/service/ampdoc-impl.js

```diff
@@ -904,6 +931,15 @@ export class AmpDocShadow extends AmpDoc {
     return this.readyPromise_;
   }
 
+  /** @override */
+  getMeta() {
+    const copy = map();
+    for (const k in this.meta_) {
```

Comment:
`Object.assign`?

Reply:
Thanks, adopted your suggestion by way of `map(opt_initial)`